c2corg_ui/static/js/map/map.js

```diff
@@ -245,6 +284,20 @@ app.MapController.DEFAULT_POINT_ZOOM = 12;
 
 
 /**
+ * @param {ol.Extent} extent Extent to recenter to.
+ * @param {olx.view.FitOptions=} options Options.
+ * @private
+ */
+app.MapController.prototype.recenterOnExtent_ = function(extent, options) {
+  var mapSize = this.map.getSize();
+  if (mapSize) {
```

Comment:
Maybe not. But is it really safe to assume this when we don't really not where this method is called?
As for now it is also used in the ``showFeatures`` function: https://github.com/c2corg/v6_ui/blob/advanced-search-ui/c2corg_ui/static/js/map/map.js#L434-L448